Excercise_3.js

```diff
@@ -1,2 +1,134 @@
 //Implement Singly Linked List.
 
+class Node {
+    constructor(val){
+        this.val = val;
+        this.next = null;
+    }
+}
+
+class SinglyLinkedList{
+    constructor(){
+        this.head = null;
+        this.tail = null;
+        this.length = 0;
+    }
+    push(val){
+        let newNode = new Node(val);
+        if(!this.head) {
+            this.head = newNode;
+            this.tail = newNode;
```

Comment:
Try forming this class with just a head pointer and performing operations based on head only.

This is verbose code